cluster/src/main/java/com/hartwig/pipeline/CommandLineOptions.java

```diff
@@ -143,13 +144,18 @@ private static Options options() {
                 .addOption(context())
                 .addOption(costCenterLabel())
                 .addOption(userLabel())
-                .addOption(targetRegionsBedLocation())
+                .addOption(useTargetRegions())
+                .addOption(targetRegionsDir())
                 .addOption(publishDbLoadEvent());
     }
 
-    private static Option targetRegionsBedLocation() {
-        return optionWithArg(TARGET_REGIONS_BED_FLAG,
-                "Specify the location of the target regions BED to enable targeted mode. Both within image and gs:// urls are supported.");
+    private static Option targetRegionsDir() {
+        return optionWithArg(TARGET_REGIONS_DIR_FLAG,
+                "Specify the location of the target regions files and enable target-regions mode. Both within image and gs:// urls are supported.");
+    }
+
+    private static Option useTargetRegions() {
```

Comment:
What's the rationale for adding an additional boolean parameter? Can we not just detect the mode via the presence of the directory?